### PR TITLE
Working v2

### PR DESCRIPTION
**Pull request by HAHOOS**
**YES**

### CODE REVIEW

File: Simple Calculator/Program.cs [hunk 1]
```diff
@@ -7,9 +7,30 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("This is a calculator. WORKING Edition by HAHOOS");
-            var math = Console.ReadLine();
-            var mathArgs = math.Split(" ");
-            Console.WriteLine((int.Parse(mathArgs[0]) + int.Parse(mathArgs[1])));
+            Console.Write("First Number:");
+            var first = int.Parse(Console.ReadLine());
+            Console.WriteLine("Operator:");
+            Console.WriteLine("1. Addition");
```
Comment (HAHOOS) [inline Simple Calculator/Program.cs:13]: Example: 1 + 1 = 2

**Review by HAHOOS — COMMENTED** [review]
LET ME IN

**Comment by ArgoTeam** [comment]
I think you can do better no money for you work

**Comment by ArgoTeam** [comment]
❌❌❌❌❌❌

**Comment by HAHOOS** [comment]
YOU CANT EVEN DO WELL YOUR PROGRAM

**Comment by ArgoTeam** [comment]
I can't Bill Gat.es learns from me

**Pull request merged**